### ISSUE TITLE
Bug: Aria-label on News & Events Page uses "Testing Label"

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/cgov-digital-platform#4598]
### Issue description
The Aria-label on the [News & Events Page](https://www.cancer.gov/news-events) is listed as "Testing Label", and that does not help visitors using aria-labels.

### Steps to reproduce the issue

1.  Go to https://www.cancer.gov/news-events
2. Inspect the date of an item in the collection
3. See that the aria-label is set to "Testing Label"

### What's the expected result?
- The aria-label for each item of the collection should have a label that is useful for screen readers.

### What's the actual result?
- The aria-label for each item of the collection is listed as "Testing Label"

### Additional details / screenshot
[- ![Screenshot]()](https://www.cancer.gov/news-events)

**Comment by olitharp-nci** [comment]
From [Accessible Name Guidance by Role](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/#naming_role_guidance)

[`list`](https://w3c.github.io/aria/#list) **Discretionary**	
* Potentially beneficial for users of screen readers that support both list names and navigation among lists on a page.
* Potentially a source of distracting or undesirable screen reader verbosity, especially if nested within a named container, such as a navigation region.
* Can be named using aria-labelledby if a visible label is present, otherwise with aria-label.

Discretionary: Naming is either optional or, in the circumstances described in the guidance column, is discouraged.

---

**Comment by laurelthrash** [comment]
To confirm during sprint planning 06/10.
